### PR TITLE
Add benchmark scores to SFT model evaluation

### PR DESCRIPTION
**Pull request by ZiyueXu77**
Fixes # .

### Description

Add scores for three benchmark tasks on SFT models trained under different settings to show the capability of FedAvg

### Types of changes

- [x] Non-breaking change (fix or new feature that would not break existing functionality).
- [ ] Breaking change (fix or new feature that would cause existing functionality to change).
- [ ] New tests added to cover the changes.
- [ ] Quick tests passed locally by running `./runtest.sh`.
- [ ] In-line docstrings updated.
- [ ] Documentation updated.

### CODE REVIEW

File: integration/nemo/examples/supervised_fine_tuning/README.md [hunk 1]
```diff
@@ -157,12 +157,25 @@ The curves shown are:
...
 - blue: alpaca
-- magenta: three datasets combined, 'centralized' training
+- magenta: three datasets combined
 
 The light curves are for single site training, and the dark curves are for FedAvg
 
-As shown, the global model from FedAvg is able to generate a loss comparable to that of centralized training.
+As shown, the global model from FedAvg is able to generate a loss comparable to that of training with all three datasets combined.
 
+Loss curve is one indication of model performance, but we also want to see if the model is able to generate reasonable text. Hence, benchmarked the trained models with standard language modeling tasks under 0-shot setting, including HellaSwag(H), PIQA(P), and WinoGrande(W).
```
Comment (holgerroth) [inline integration/nemo/examples/supervised_fine_tuning/README.md:166]: "Hence, benchmarked" -> "Hence, we benchmarked"
"0-shot" -> "zero-shot".

Also, can you add links to the zeros shot task datasets?

Comment (ZiyueXu77) [reply integration/nemo/examples/supervised_fine_tuning/README.md]: Thanks! Added

**Pull request merged**